public/index.html

```diff
@@ -100,11 +113,22 @@ <h2 class="article-title">{{ title }}</h2>
     <section id="projects" class="tab-content">
       <h1>Student Projects</h1>
       <div id="projectContainer">
```

Comment:
This section looks like it could be using a template and sourcing the data from somewhere, similar to the blog....